packages/dashboard-message-bus/lib/message/types.ts

```diff
@@ -31,6 +31,16 @@ export interface LogMessage extends Message {
   };
 }
 
+/**
+ * Message to log in Dashboard browser console
+ */
+export interface DebugMessage extends Message {
+  type: "debug";
```

Comment:
Perhaps it's time to refine the Message interface to keep track of valid types. @gnidan ?

```typescript

type MessageType = "debug" | "provider" | "log" // | etc...
export interface Message {
  id: number;
  type: MessageType;
  payload: any;
}
```